[FIX] CD 파이프라인 에러 해결

## ⚙️ ISSUE
- CD 빌드 과정에서 다음과 같은 에러가 발생합니다.
```
CourseXApplicationTests > contextLoads() FAILED
    java.lang.IllegalStateException at DefaultCacheAwareContextLoaderDelegate.java:180
        Caused by: java.lang.IllegalStateException at TestPropertySourceUtils.java:317
            Caused by: java.io.FileNotFoundException at ClassPathResource.java:215

1 test completed, 1 failed

FAILURE: Build failed with an exception.
```

--- a/.github/workflows/main-CD.yml
+++ b/.github/workflows/main-CD.yml
@@ -23,7 +23,7 @@ jobs:
       - name: Create resources folder and add env file
         run: |
           mkdir -p courseX-backend/src/main/resources/env
-          echo "${{ secrets.ENV_FILE }}" > courseX-backend/src/main/resources/env/prod.env
+          echo "${{ secrets.PROD_ENV_FILE }}" > courseX-backend/src/main/resources/env/prod.env
 
       - name: Build backend
         run: |
@@ -84,7 +84,7 @@ jobs:
             mv docker-compose.processed.yml docker-compose.yml
             mv deploy.processed.sh deploy.sh
             chmod +x deploy.sh
-            echo "${{ secrets.ENV_FILE }}" > .env
+            echo "${{ secrets.PROD_ENV_FILE }}" > .env
             export ECR_REGISTRY=${{ steps.login-ecr.outputs.registry }}
             export IMAGE_TAG=${{ github.sha }}
             ./deploy.sh

